stories/assets/js/navigation.js

```diff
@@ -57,7 +57,13 @@ export const navigationInitialize = (locale) => {
   });
 
   $('.mega-wrapper').mouseleave(function () {
-    $(this).find('.mega-nav-option.show-mega').removeClass('show-mega');
+    $(this).find('.mega-nav-option.show-mega').removeClass('show-mega').addClass('hide');
+    setTimeout(function () {
+      $('.mega-nav-option').removeClass('hide').addClass('show-mega-back');
+    }, 10);
+    setTimeout(function () {
```

Comment:
@shishirphadke before menu close we need time to play back animation. So that we have added timeout there. 
and for two times of use it. We need to make first opening animation none first start the next animation.

Reply:
@shishirphadke done.